translations/binaries/slps_strings_title.json

```
@@ -803,7 +803,7 @@
             "string": "主婦として必要な事、[LINE]\nそれは近所付き合いや井戸端会議への列席。[LINE]\nという場合もあるにはあるらしい。[END]",
             "original_bytes_length": 95,
             "available_bytes_size": 95,
-            "translation": "주부로서 필요한 것,[LINE]\n그것은 이웃과의 관계 및 웰빙 회의에 참석하는 것.[LINE]\n그런 경우도 있다고 한다.[END]"
+            "translation": "주부에게 필요한 일은[LINE]\n이웃과 친해지는 것과 아줌마 수다 떨기.[LINE]\n라고 하는 경우도 있다고 한다.[END]"
```

```suggestion
            "translation": "주부에게 필요한 일은[LINE]\n이웃과 친해지는 것과 수다 떨기.[LINE]\n라고 하는 경우도 있다고 한다.[END]"
```

이미 앞에서 '주부', '이웃'이라는 맥락이 포함되어있으므로 원문('井戸端会議')의 의미를 똑같이 표현할 수 없다면 그냥 '수다 떨기'라고 해도 얼추 의미가 맞게 전달되는 것 같습니다~